src/utils.ts

```diff
@@ -92,8 +93,12 @@ export const formatDateTime = (now: Date, timezone: 'local' | 'utc' = 'local') =
 };
 
 export async function getFileContent(file: TFile): Promise<string | null> {
-  if (file.extension != "md") return null;
-  return await this.app.vault.read(file);
+  if (file.extension == "md") {
+    return await this.app.vault.read(file);
+  } else if (file.extension == "pdf") {
```

Comment:
Does this mean this PR works only for a PDF file opened directly in Obsidian, and not embedded in another markdown note? I initially misunderstood and tried an _embedded_ PDF, and it didn't work. Don't get me wrong, this is still phenomenal work!! Just wondering, after your research, do you think there's a path to enable these features for embedded PDFs in a markdown file?